src/server/Listener.js

```diff
@@ -151,4 +157,12 @@ function createServer (port, app) {
 	return server;
 }
 
+function createMock (uri) {
+	let keys = []; // eslint-disable-line prefer-const
+	return {
+		keys: keys,
```

Comment:
Ahh, after reading the documentation I can see that keys are populated by the module.